profiles/rgw-radosgw-static-website

```diff
@@ -0,0 +1,11 @@
+---
+# THIS FILE IS AN EXAMPLE THAT CONTAINS A SET OF VARIABLE FOR A PARTICULAR PURPOSE
+# GOAL: CONFIGURE RADOS GATEWAY WITH STATIC WEBSITE
+#
+# The following variables should be added in your group_vars/rgws file
+# The double quotes are important, do NOT remove them.
+
+ceph_conf_overrides:
+  "client.rgw.{{ hostvars[inventory_hostname]['ansible_hostname'] }}":
+    rgw enable static website = {{ radosgw_static_website }}
```

Comment:
Don't use a variable to assign the content. Set proper values as an example.
e.g: `rgw enable static website = true`. Same for all the variables.